spec/web_human_player_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "web_human_player"
+
+RSpec.describe WebHumanPlayer do
+  let(:board_stub) { instance_double(TicTacToe::Board) }
+  let(:display) { WebDisplay.new }
+  let(:web_human) { WebHumanPlayer.new(:X, display) }
+
+  it "doesn't currently have a move available" do
+    expect(web_human.has_new_move?).to eq(false)
+  end
+
+  it "display doesn't have a new move" do
+    expect(display).to receive(:has_new_move?).and_return(false)
```

Comment:
prefer stub here, shoot for a single (logical) assertion per test. each expect is an assertion.